### ISSUE TITLE
No issues since V1.9.3

### ISSUE THREAD

**Issue by movedoa** [Riverside-Software/sonar-openedge#37]
There are no issues found since i updated to 1.9.3.
The analysis runs normally but i see no issues in sonar.

The only thing in the logfile that look suspicious are the following lines:
```
[sonar:sonar] JaCoCoSensor: JaCoCo report not found : C:\Buildtools\Jenkins\workspace\Sonar_Runner\target\jacoco.exec
[sonar:sonar] JaCoCoItSensor: JaCoCo IT report not found: C:\Buildtools\Jenkins\workspace\Sonar_Runner\target\jacoco-it.exec
.....
[sonar:sonar] Invalid licence - Signature KO - Expiration OK
```

**Comment by gquerret** [comment]
Do you use version 1.0.6 of riverside-rules-plugin.jar ? The licence for the rules was updated here.

**Comment by movedoa** [comment]
Yes, Sonar shows 1.0.6 installed in the update center

**Comment by gquerret** [comment]
I'll double-check if the licence is correct in this file. Will keep you posted this afternoon.

**Comment by gquerret** [comment]
The licence wasn't signed correctly in riverside-rules-1.0.6.jar. I've overwritten the JAR file on the releases page, so please update it.

**Comment by gquerret** [comment]
I'll also contact you offline by the end of the week on how to get new evaluation and commercial licenses.
